core/src/main/java/org/mule/runtime/core/internal/management/stats/PayloadStatisticsList.java

```diff
@@ -14,10 +19,21 @@
 
 import org.apache.commons.collections.list.AbstractListDecorator;
 
-final class PayloadStatisticsList<T> extends AbstractListDecorator {
+final class PayloadStatisticsList<T> extends AbstractListDecorator implements Externalizable {
+
+  private static final long serialVersionUID = -5991764233175594789L;
 
   private final LongConsumer populator;
 
+  public PayloadStatisticsList() {
```

Comment:
would this be needed if there were a kryo serializer for this class?

Reply:
the void constructor is needed for non-kryo scenarios. Kryo also needs all parameters to be initialized in the void constructor.